codec-http/src/test/java/io/netty5/handler/codec/http/HttpClientCodecTest.java

```diff
@@ -322,27 +321,24 @@ public void testDecodesFinalResponseAfterSwitchingProtocols() {
     }
 
     @Test
-    public void testWebSocket00Response() {
-        byte[] data = ("HTTP/1.1 101 WebSocket Protocol Handshake\r\n" +
-                "Upgrade: WebSocket\r\n" +
-                "Connection: Upgrade\r\n" +
-                "Sec-WebSocket-Origin: http://localhost:8080\r\n" +
-                "Sec-WebSocket-Location: ws://localhost/some/path\r\n" +
-                "\r\n" +
-                "1234567812345678").getBytes();
+    public void testWebSocketResponse() {
+        byte[] data = ("HTTP/1.1 101 Switching Protocols\r\n" +
+                       "Upgrade: websocket\r\n" +
+                       "Connection: Upgrade\r\n" +
+                       "Sec-WebSocket-Accept: s3pPLMBiTxaQ9kYGzzhZRbK+xOo= \r\n" +
+                       "Sec-WebSocket-Protocol: chat\r\n\r\n")
+                .getBytes(CharsetUtil.US_ASCII);
         EmbeddedChannel ch = new EmbeddedChannel(new HttpClientCodec());
         assertTrue(ch.writeInbound(ch.bufferAllocator().copyOf(data)));
 
         HttpResponse res = ch.readInbound();
```

Comment:
`HttpResponse` does not itself have a `close()` method, but pretty much everything that implements it does.
I think it'd be prudent to pass it to `Resource.dispose()` when we're done with it.

Reply:
Let's, add